### DISCUSSION TITLE
Sonoff 4ch pro r2 Force AP Mode

### DISCUSSION THREAD

**Discussion by pranavptl** [arendst/Tasmota#15124] [Q&A] [answered]
Hi,
Lost my access to a sonoff 4ch pro r2 and needed to reset the AP - any way to do it with a series of button presses? I dont want to reflash...

**Comment by pranavptl** [comment, marked as answer]
sorry pretty sure i hadnt updated the tasmota from an old version

**Reply by barbudor** [reply]
Every recovery options are in the doc
https://tasmota.github.io/docs/Device-Recovery/
